build/yamls/antrea.yml

```diff
@@ -4397,9 +4403,12 @@ rules:
       - networkpolicystats
       - antreaclusternetworkpolicystats
       - antreanetworkpolicystats
+      - nodeiplatencystats
     verbs:
       - get
       - list
+      - create
+      - update
```

Comment:
I don't think you regenerated these files correctly? This change has been removed from the Helm chart templates.

Reply:
Ok, I have regenerated them.